test/integration/glue_test.py

```diff
@@ -222,14 +241,16 @@ def test_unmapped_utils_eia(
 ):
     """Check for unmapped EIA Plants.
 
-    This test is duplicative with the sql foriegn key constraints.
+    This test has its own call signature because its more complex. In order to label the
+    missing utilities, we use both the ``pudl_out`` object as well as direct SQL
+    queries. This test is duplicative with the sql foriegn key constraints.
     """
-    unmapped_utils_eia = get_unmapped_utils_eia(
+    unmapped_utils_eia = get_utils_ids_eia_unmapped(
```

Comment:
I think you updated the equivalent FERC function to be `get_util_ids...`. Probably both the EIA and FERC functions should have similar names. (singular utils, plural IDs)

Reply:
yes your right!